src/js/actions/libraries.js

```diff
@@ -128,7 +128,8 @@ define(function (require, exports) {
      * @type {Set}
      */
     var EDITABLE_GRAPHIC_REPRESENTATION_TYPES = new Set([
-        REP_PHOTOSHOP_TYPE
+        REP_PHOTOSHOP_TYPE,
+        "application/photoshop.large" // PSB
```

Comment:
Where is REP_PHOTOSHOP_TYPE defined? can you define a constant for `application/photoshop.large` in the same place?